### ISSUE TITLE
the principle of new Date(),new Date([1]),new Date([-1])?

### ISSUE THREAD

**Issue by BainDragneel** [nodejs/help#2246]
* **Node.js Version**: v12.12.0
* **OS**: Mac OS 10.14.4
* **Scope (install, code, runtime, meta, other?)**: other
* **Module (and version) (if relevant)**:

hi,I have some problems, I hope someone will tell me.
i want to know the principle of new Date(). 
When i run new Date([1]),it returns "2000-12-31T16:00:00.000Z".
When i run new Date([-1]),it returns"2000-12-31T16:00:00.000Z".
i want to know why?

**Comment by bnoordhuis** [comment]
Is your question why you get the same date for different inputs?

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date. The array you provide as input is stringified so what you're actually passing in are the strings `'1'` and `'-1'`.

Those aren't valid date strings and while the following caveat from the documentation applies in general, it applies even more for invalid inputs:

> Note: Parsing of date strings with the Date constructor (and Date.parse(), which works the same way) is strongly discouraged due to browser differences and inconsistencies. 

Basically, the JS engine needs to do _something_ but not necessarily something that makes sense.

**Comment by tniessen** [comment]
No follow-up from @BainDragneel, closing as answered.
